Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/exchangemgr/ExchangeManager.java

```diff
@@ -84,14 +87,10 @@ private void loadExchangeInfo() throws ExchangeManagerException {
                 exCache.clear();
                 if (null != exInfo.getExchanges() && CollectionUtils.isNotEmpty(exInfo.getExchanges().getExchange())) {
                     for (ExchangeType ex : exInfo.getExchanges().getExchange()) {
-                        if (null != ex.getOrganizationList()
-                            && CollectionUtils.isNotEmpty(ex.getOrganizationList().getOrganization())
-                            && StringUtils.isNotEmpty(ex.getType()) && StringUtils.isNotEmpty(ex.getName())) {
-                            Map<String, OrganizationType> innerMap = new HashMap<>();
-                            for (OrganizationType org : ex.getOrganizationList().getOrganization()) {
-                                innerMap.put(org.getHcid(), org);
-                            }
-                            exCache.put(ex.getName(), innerMap);
+                        if (isOverrideExchange(ex)) {
+                            overrideExchange = ex;
```

Comment:
Do we need to break here? What is the requirement / possibility of having multiple overrides defined? as it stands if we have more than one, the latest will stick and the previous ones will overwritten.

Reply:
We discussed to have only one Override exchange type in exchangeInfo.xml.  ExchangeName is not important for an override exchange type  so  we cannot distinguish even if the user provides more than one override exchange. 
As discussed offline, i will add a warning when there are more than one override exchange type and the previous one is about to overwritten.